### PR TITLE
Feature/matchmaking

### PR DESCRIPTION
**Pull request by whitewind664**

### CODE REVIEW

File: app/src/main/java/com/github/gogetters/letsgo/activities/OnlineGameActivity.kt [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package com.github.gogetters.letsgo.activities
+
+import android.os.Bundle
+import android.widget.TextView
+import androidx.appcompat.app.AppCompatActivity
+import com.github.gogetters.letsgo.R
+
+class OnlineGameActivity: AppCompatActivity() {
```
Comment (sirpudika) [inline app/src/main/java/com/github/gogetters/letsgo/activities/OnlineGameActivity.kt:8]: Write a stupid test for this.

**Review by sirpudika — APPROVED** [review]
Good implementation to the UI :)

**Review by smikulas — APPROVED** [review]

File: app/src/main/java/com/github/gogetters/letsgo/database/Database.kt [hunk 2]
```diff
@@ -69,6 +69,8 @@ class Database {
                             currentData.child("currentlyWaiting").value = null
                             val gameId = database.child("matchmaking").child("games").push().key
                             currentData.child("games").child("$gameId").value = gameData
+                            currentData.child("currentGamesPerUser").child(gameData.player1).value = gameId
+                            currentData.child("currentGamesPerUser").child(gameData.player2).value = gameId
                         }
```
Comment (lupusmalus) [inline app/src/main/java/com/github/gogetters/letsgo/database/Database.kt:98]: store these strings in constants

**Review by lupusmalus — APPROVED** [review]
LGTM, but add documentation on your code

**Comment by whitewind664** [comment]
Probably shouldn't merge this one for tomorrow, gonna finish it next week

**Pull request merged**